docs/content/preview/launch-and-manage/monitor-and-alert/xcluster-monitor.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Monitor xCluster
+headerTitle: Monitor xCluster
+linkTitle: xCluster
+description: Monitoring the health of xCluster replication
+headContent: Monitoring the state and health of xCluster replication
+menu:
+  preview:
+    parent: monitor-and-alert
+    identifier: xcluster-monitor
+    weight: 110
+type: docs
+---
+
+## Metrics
+
+The list of xCluster metrics is available in the [xCluster metrics](../metrics/replication/).
+
+## Console
```

Comment:
do we want to expose instructions for looking at `ddl_queue` or `replicated_ddls` in docs?